### DISCUSSION TITLE
Welcome to HorseExtension Discussions!

### DISCUSSION THREAD

**Discussion by sashaman** [sashaman/HorseExtension#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guiibz** [comment]
Hello, I'm Guilherme, from Brazil

I followed the steps in this tutorial to change No Man's Land files in farming simulator 19 and was successful. Thank you for this help. 

Now I would really like to perform the steps of this tutorial in no man's land (NML) looking to add the Alfalfa, clover, rye, spelt and triticale crops and the Maize plus functionalities in full, however, in farming simulator 22.

My question is, is it possible? Is there any specific tutorial for this purpose? 

I'm currently scouring the internet looking for information that could help me with this process.

Thanks again for all your help in updating the files in farming simulator 19.
